src/app.js

```diff
@@ -579,15 +576,15 @@ fluid.defaults("gpii.app.menu", {
             singleTransform: {
                 type: "fluid.transforms.free",
                 func: "gpii.app.menu.generateMenuTemplate",
-                args: ["{that}.model.keyedInUser", "{that}.model.keyOut", "{that}.options.snapsets", "{that}.options.exit"]
+                args: ["{that}.model.keyedInUser", "{that}.options.snapsets", "{that}.model.keyOut", "{that}.options.exit"]
             },
             priority: "last"
         }
     },
     menuLabels: {
         keyedIn: "Keyed in with %userTokenName",    // string template
-        keyOut: "Key out %userTokenName",           // string template
-        notKeyedIn: "Not keyed in",
+        keyOut: "Key-out of GPII",
```

Comment:
This doesn't agree with the wording in the wireframes at https://issues.gpii.net/browse/GPII-2527?jql=project%20%3D%20GPII%20AND%20component%20%3D%20%22Personal%20Control%20Panel%22 -